posthog/api/test/test_organization.py

```diff
@@ -69,10 +73,20 @@ def test_rename_organization_if_admin(self):
         self.organization_membership.level = OrganizationMembership.Level.ADMIN
         self.organization_membership.save()
         response = self.client.patch(f"/api/organizations/{self.organization.id}", {"name": "QWERTY"})
-        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.organization.refresh_from_db()
         self.assertEqual(self.organization.name, "QWERTY")
 
+    def test_update_domain_whitelist_if_admin(self):
+        self.organization_membership.level = OrganizationMembership.Level.ADMIN
+        self.organization_membership.save()
+        response = self.client.patch(
+            f"/api/organizations/{self.organization.id}", {"domain_whitelist": ["posthog.com", "movies.posthog.com"]}
+        )
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.organization.refresh_from_db()
+        self.assertEqual(self.organization.domain_whitelist, ["posthog.com", "movies.posthog.com"])
+
```

Comment:
Could there be a point in testing we also can't do that if we're not an admin?

Reply:
Well actually the test was misleading, in theory any team member could update this, as any team member can invite another team member, but after more thought I do agree this is more of an admin config. Will update & add the relevant test accordingly.